app/views/assessment/AssessmentEndDistancing.js

```diff
@@ -0,0 +1,31 @@
+import React, { useContext } from 'react';
```

Comment:
Could we make a folder under assessments for all of these end screens? E.g. `app/views/assessment/endScreens/AssessmentEndDistancing.js`